Octokit/Http/ProductHeaderValue.cs

```diff
@@ -38,9 +40,16 @@ public ProductHeaderValue(string name, string version)
         {
         }
 
-        ProductHeaderValue(System.Net.Http.Headers.ProductHeaderValue productHeader)
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ProductHeaderValue"/> class.
+        /// </summary>
+        /// <remarks>
+        /// See more information regarding User-Agent requirements here: https://developer.github.com/v3/#user-agent-required
+        /// </remarks>
+        /// <param name="name">The <see cref="System.Net.Http.Headers.ProductHeaderValue"/>.</param>
+        public ProductHeaderValue(System.Net.Http.Headers.ProductHeaderValue productHeader)
```

Comment:
Is this the meaningful part of the change? I'm having a hard time figuring out how the internal _productHeaderValue was set before the change.

Reply:
The changing of the constructor from private to public is the meaningful bit yeah. All other constructors call it.